service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/disk/ControlledAzureDiskHandler.java

```diff
@@ -0,0 +1,19 @@
+package bio.terra.workspace.service.resource.controlled.cloud.azure.disk;
+
+import bio.terra.workspace.db.model.DbResource;
+import bio.terra.workspace.service.resource.model.WsmResource;
+import bio.terra.workspace.service.resource.model.WsmResourceHandler;
+import java.util.Optional;
+
```

Comment:
documentation for public class

Reply:
Do I need that if the interface class provides the documentation?

Reply:
[/** {@inheritDoc} */] i think you can put this

Reply:
Perfect! I did that.